src/components/Profile/Profile.js

```diff
@@ -50,9 +51,42 @@ class UserProfile extends Component {
       }));
     }
   }
+
+  handleSubmit = e => {
+    e.preventDefault();
+    console.log("submitt");
+    if ( this.state.newpassword!=="" && this.state.confnewpassword==="" ) {
+      alert("Please confirm your password !!!");
```

Comment:
Don't use browser alert set it in a state called message and display it below the text field if any error